docs/source/api/algorithms/ensembles/decision-forest.rst

```diff
@@ -92,6 +92,18 @@ voting_mode::weighted
 voting_mode::unweighted
    The final prediction is combined through a simple majority voting.
 
+`splitter_mode`
+~~~~~~~~~~~~~~~
+
+splitter_mode::best
+   Best splitting strategy chooses best treshold for each feature while building trees 
+   in terms of impurity among all histogram bins and feature subset.
```

Comment:
```suggestion
   in terms of impurity among all histogram bins and feature subsets.
```

Reply:
Maybe use something: The best splitting strategy is an important part of the decision tree learning algorithm. It aims to find the best split for each feature that results in the largest reduction in impurity of the target variable.